internal/tasks/publish-index-image-task/tests/test-publish-index-image-fail.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-publish-index-image-fail
+  annotations:
+    test/assert-task-failure: "run-task"
```

Comment:
Why does this fail? Looking at the task script, it has: `skopeo && echo success || echo failed`, but the task itself should succeed even if the skopeo command fails, no?

Reply:
added more description

Reply:
Ah, right, sorry, somehow I misread the logic with grep - I saw it recently and understood how it works, but forgot again :) Thanks.